xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -14875,6 +14880,15 @@ glusterd_check_brick_order(dict_t *dict, char *err_str, int32_t type,
     ai_list->info = NULL;
     CDS_INIT_LIST_HEAD(&ai_list->list);
 
+    pre_list = MALLOC(sizeof(addrinfo_list_t));
```

Comment:
>>> Any specific reason to use MALLOC instead of using GF_MALLOC

No specific reason, just to keep the new code inline with the code present, i chose to use MALLOC as it was used substantially in this function